packages/oruga/src/components/select/Select.vue

```diff
@@ -230,7 +230,7 @@ const { parentField, statusVariant, statusVariantIcon } = injectField();
 
 const vmodel = defineModel<T | T[]>({ default: undefined });
 
-const placeholderVisible = computed(() => vmodel.value === null);
+const placeholderVisible = computed(() => !vmodel.value);
```

Comment:
I think this may technically be a change in behavior if the `v-model` value is zero.

Reply:
Or you are right! Thank you! I will change this to be more explicit.